README.md

```diff
@@ -26,52 +26,52 @@ Building with node-gyp (via node-pre-gyp) requires:
 
 ## Installation
 
-The `appoptics-apm` module is [available on npm](http://npmjs.org/package/appoptics-apm) and can be installed
+The `solarwinds-apm` module is [available on npm](http://npmjs.org/package/solarwinds-apm) and can be installed
 by navigating to your app root and running:
 
 ```
-npm install --save appoptics-apm
+npm install --save solarwinds-apm
 ```
 
-The agent requires a service key, obtained from the AppOptics dashboard under "Organization Details",
-to connect to your account.  This is set via the `APPOPTICS_SERVICE_KEY` environment variable, make
+The agent requires a service key, obtained from the SolarWinds dashboard under "Organization Details",
+to connect to your account.  This is set via the `SW_APM_SERVICE_KEY` environment variable, make
 sure it is available in the environment where your application is running:
 
 ```
-export APPOPTICS_SERVICE_KEY="api-token-here:your-service-name"
+export SW_APM_SERVICE_KEY="api-token-here:your-service-name"
 ```
 
 Then, at the top of your main js file for your app, add this:
 
 ```
-require('appoptics-apm')
+require('solarwinds-apm')
 ```
 
-Now restart your app and you should see data in your AppOptics dashboard in a minute or two.
+Now restart your app and you should see data in your SolarWinds dashboard in a minute or two.
 
 ## Important!
 
-`appoptics-apm` should be the first file required. If, for example, you are using the `esm`
+`solarwinds-apm` should be the first file required. If, for example, you are using the `esm`
 package to enable ES module syntax (import rather than require) and you use the following
 command to invoke your program `node -r esm index.js` then `esm.js` is loaded first and
-`appoptics-apm` is unable to instrument modules. You can use it, just make sure to require
-`appoptics-apm` first, e.g., `node -r appoptics-apm -r esm index.js`.
+`solarwinds-apm` is unable to instrument modules. You can use it, just make sure to require
+`solarwinds-apm` first, e.g., `node -r solarwinds-apm -r esm index.js`.
 
-If you are using the custom instrumentation SDK then appoptics must be loaded in the code
-so that a reference to the SDK is obtained, like `const ao = require('appoptics-apm')`. It
-is still be possible to use the command line `node -r appoptics-apm -r esm index.js`; the
+If you are using the custom instrumentation SDK then SolarWinds APM must be loaded in the code
+so that a reference to the SDK is obtained, like `const ao = require('solarwinds-apm')`. It
```

Comment:
If docs nor lambda aren't in the scope of this Super PR then please ignore this one!
There are mentions of `const ao = require('appoptics-apm')`  in these files on this branch, when I think it should all be `const ao = require('solarwinds-apm')` unless this is still how it works in current state:

* `docs/api.md`
* `docs/lambda-configuration.md`
* `docs/migrating.md`
* `lib/api.js`

Reply:
The lambda implementation suffers from several issues and needs fixing. Work has been mapped into tickets a while back but has been "deprioritized". Thus, for SolarWinds APM the lambda functionality has been disabled. 

In order to give future maintainers a fighting chance in reenabling this thing, code has not been removed and was deliberately kept "as-is" so that git history is not "polluted" and work can be done in either repo. (FWIW, I personally would have done anything that falls under "fix", including lambda fixing, in AppOptics first).